.golangci.yml

```diff
@@ -0,0 +1,62 @@
+# This file contains all available configuration options
+# with their default values.
+
+# options for analysis running
+run:
+  # default concurrency is a available CPU number
+  concurrency: 4
+
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 2m
+
+  # exit code when at least one issue was found, default is 1
+  issues-exit-code: 1
+
+  # include test files or not, default is true
+  tests: true
+
+
+linters-settings:
+  lll:
+    line-length: 130
+
+linters:
+  enable:
+    - deadcode
+    - goimports
+    - golint
+    - ineffassign
+    - interfacer
+    - lll
+    - misspell
+    - unconvert
+    - unparam
+    - vet
```

Comment:
It seems like this cant be run. When I look at the logs I see this:

`[33mWARN[0m [runner] Can't run linter govet: asmdecl: analysis skipped due to errors in package `

I got this locally too.

Reply:
Excluded govet for now from linters